### PR TITLE
test: Add helper function for generating blocks for p2p_syncing_test.py

### PR DESCRIPTION
**Pull request by altonen**

### CODE REVIEW

File: chainstate/src/detail/tests/mod.rs [hunk 1]
```diff
@@ -162,3 +164,27 @@ fn create_new_outputs(tx: &Transaction) -> Vec<(TxInput, TxOutput)> {
         .filter_map(move |(index, output)| create_utxo_data(&tx.get_id(), index, output))
         .collect::<Vec<(TxInput, TxOutput)>>()
 }
+
+// generate 5 regtest blocks and print them in hex
+#[ignore]
+#[test]
+fn generate_blocks_for_functional_tests() {
```
Comment (TheQuantumPhysicist) [inline chainstate/src/detail/tests/mod.rs:171]: Please add a comment that this should be removed when block production is complete and is usable through RPC

**Review by TheQuantumPhysicist — APPROVED** [review]

File: chainstate/src/detail/tests/mod.rs (hunk 1 above)
Comment (TheQuantumPhysicist) [reply chainstate/src/detail/tests/mod.rs:171]: Another TODO :-)

Comment (altonen) [reply chainstate/src/detail/tests/mod.rs:171]: For each item I remove the TODO list, I add 1-3 more. I need to start working *really* fast :smiley:

**Review by iljakuklic — APPROVED** [review]
Could this be an example rather than an ignored test? Seems a better fit for this use case to me but an example would not have the access to private methods.

**Comment by altonen** [comment]
I like the idea of making it an example but it uses private modules so I don't know if we want to expose them just for this example.

**Review by muursh — APPROVED** [review]

**Pull request merged**